### PR TITLE
Updating notes for current conditions

### PR DESCRIPTION
**Pull request by aforbis-stokes**

### CODE REVIEW

File: software/cloud-dashboard/react-client/src/components/temp-view/notes/notes-data.js [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
-	"Duke": "Not in operation while a new circuit is under development.",
-	"Philippines-Sagay": "",
-	"Philippines-Subayon": "", 
-	"India": "",
+	"Kenya-Central": "Temporary decrease in location's occupancy. Outages due to biogas being conserved.", 
+	"Kenya-North": "",
+	"Duke": "Microcontroller for testing purposes only.",
+	"Philippines-Sagay": "No cellular connectivity in this region. Data transmission under development.",
+	"Philippines-Subayon": "No cellular connectivity in this region. Data transmission under development.", 
+	"India": "Heating system temporarily not it operation. Digester health being monitored.",
```
Comment (suyashkumar) [inline software/cloud-dashboard/react-client/src/components/temp-view/notes/notes-data.js]: looks like there's a small spelling mistake: "temporarily not **it** operation."

**Review by suyashkumar — DISMISSED** [review]
Small spelling mistake. I can fix and merge.

**Review by suyashkumar — APPROVED** [review]
Fixed spelling mistake.

**Comment by mlp6** [comment]
@aforbis-stokes I just "assigned" it to you; feel free to merge it in using the 'Squash-n-Merge' option.  (Squashing just compresses multiple discrete commits into one logical one on the master branch.)

**Pull request merged**